src/main/java/racingcar/model/RacingGame.java

```diff
@@ -0,0 +1,42 @@
+package racingcar.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RacingGame {
+    private final List<Car> cars;
+    private final int rounds;
+    private int currentRound;
+
+    public RacingGame(List<Car> cars, int rounds) {
+        this.cars = cars;
+        this.rounds = rounds;
+        this.currentRound = 0;
+    }
+
+    public boolean hasNextRound() {
+        return currentRound < rounds;
+    }
+
+    public void playRound() {
+        for (Car car : cars) {
+            car.move();
+        }
+        currentRound++;
```

Comment:
전위 연산자를 이용하는게 후위 연산자를 이용하는것보다 성능이 좋답니다😃
```suggestion
       ++currentRound;
```